src/components/OutdoorPage.js

```diff
@@ -4,13 +4,56 @@ import { ReactComponent as RightArrow } from "../icons/arrow-right-thin.svg";
 import { ReactComponent as LeftArrow } from "../icons/arrow-left-thin.svg";
 import { ReactComponent as Progress } from "../icons/outdoor-progress.svg";
 import { ReactComponent as Sun } from "../icons/sun.svg";
+import { ReactComponent as Cloudy } from "../icons/sun.svg";
+import { ReactComponent as PartlyCloudy } from "../icons/cloud-sun.svg";
+import { ReactComponent as Rain } from "../icons/cloud-rain.svg";
+import { ReactComponent as Thunder } from "../icons/cloud-lightning.svg";
 import { Link, useHistory, useLocation } from "react-router-dom";
+import { useEffect, useState } from "react";
 import { stringify, parse } from "../utilities/queryString";
 import { toggleValueInArray } from "../models/toggleValueInArray";
 
 export default function OutdoorPage({ restaurantData }) {
   const location = useLocation();
   const history = useHistory();
+  const [weather, setWeather] = useState();
+
+  useEffect(() => {
+    const url =
+      "https://api.weatherapi.com/v1/current.json?key=487e8af21edb4a54b1f142156211105&q=Hamburg&aqi=no";
+
+    fetch(url)
+      .then((res) => res.json())
+      .then((incomingData) => {
+        const newWeather = incomingData.current;
+        setWeather(newWeather);
+      })
+      .catch((error) => {
+        console.log(error);
+      });
+  }, []);
+
+  function showWeatherIcon() {
+    if (weather !== undefined) {
+      if (weather.condition.text.includes("Partly cloudy")) {
+        return <PartlyCloudy className="outdoor-weather__svg" />;
+      } else if (weather.condition.text.includes("sunny")) {
+        return <Sun className="outdoor-weather__svg" />;
+      } else if (weather.condition.text.includes("rain")) {
+        return <Rain className="outdoor-weather__svg" />;
+      } else if (weather.condition.text.includes("Overcast")) {
+        return <Cloudy className="outdoor-weather__svg" />;
+      } else if (weather.condition.text.includes("thunder" || "lightning")) {
+        return <Thunder className="outdoor-weather__svg" />;
+      }
+    }
+  }
+
+  function showTemp() {
+    if (weather !== undefined) {
+      return weather.temp_c;
+    }
+  }
```

Comment:
hey @soerendolling 
In my opinion, this whole weather stuff should be its own component (maybe something like `WeatherWidget`).
Th Outdoorpage should just use it

Reply:
showWeatherIcon() looks like a neat way to implement a switch-statement